tests/contrib/network/test_ufw.py

```diff
@@ -531,12 +531,8 @@ def test_reload_fail(self, modprobe, check_output, log):
         log.assert_any_call("ufw couldn't be reloaded", level='WARN')
 
     def test_status(self):
-        if six.PY2:
-            return
-        with mock.patch('subprocess.run') as run:
-            cp = mock.MagicMock()
-            cp.stdout = UFW_STATUS_NUMBERED
-            run.return_value = cp
+        with mock.patch('subprocess.check_output') as check_output:
```

Comment:
@fnordahl @ajkavanagh -- can you confirm this a better fix?

Reply:
More concerned with losing PY2 support from the library.

Reply:
this actually includes py2 support rather than losing it.

Reply:
That's good then!